src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -142,9 +143,6 @@ func Run(updateHandler func(updated *Config)) error {
 
 func updateGlobals(cfg *Config) error {
 	globals.InstanceId = cfg.InstanceId
-	loc := &geolookup.City{}
-	loc.Country.IsoCode = cfg.Country
-	globals.SetLocation(loc)
```

Comment:
I'm not sure why this was being done, but I think this was done on purpose.   Actually, after removing this I don't think there is any point to having a country in the config anymore.

Plausible reasons:

- allow setting geolocation for testing stuff

- save location across runs so the user dot doesn't start in null island until first geolocation request.

Since none of these looks too compelling, I'm fine with removing these lines *and* cfg.Country.